src/config/config.yml

```diff
@@ -16,7 +16,7 @@ targets:
   test-results:
     queue: test-results
     dlQueue: cvs-edh-dispatcher-test-results-${BRANCH}-dlq
-    swaggerSpecFile: API_Vehicle_Testresults_28.1.yaml
+    swaggerSpecFile: API_Vehicle_Test_Results_CVS->EDH_v1.yaml
```

Comment:
Linux seems to accept angled brackets in filenames but I would generally suggest to avoid them.

Reply:
I was using them to make the purpose of the file clear. I hadn't considered that they might be special characters, but nothing complained so I think we're ok. Might complain on Windows.... we'll cross that bridge when we get to it I guess